GunShipAlpha/src/Behemoth.java

```diff
@@ -0,0 +1,94 @@
+import java.awt.*;
+import java.awt.geom.Point2D;
+import java.util.Random;
+import java.util.Timer;
+import java.util.TimerTask;
```

Comment:
nope, tank and behemoth are identical, or nearly so. you should have guessed that i'd immediately say no to that. 

if you need a pointer on how to fix it, look at "inheritance". There's a few other ways, as well, but inheritance from a common ancestor is probably the easiest.

Reply:
also, please don't submit pull requests with duplication like this - as i said, you should _know_ instinctively that this is wrong. if you don't know how to fix it, throw up a hand and ask what to do, but it makes me feel like I haven't taught you the most basic thing about duplication when you submit a PR like this :)

Reply:
that's fair, I'll fix it